py3status/modules/do_not_disturb.py

```diff
@@ -10,7 +10,7 @@
     pause: specify whether to pause or kill processes; for dunst
         see `Dunst Miscellaneous` section for more information
         (default False)
-    server: specify server to use, eg dunst or xfce4-notifyd, otherwise auto
+    server: specify server to use, eg dunst, xfce4-notifyd or mako, otherwise auto
```

Comment:
This should say `mako, dunst, xfce4-notifyd`. Order of servers kinda matter too. Sorry for too many reviews.

Reply:
We thank you for your reviews, do not apologize :wink: Agree that consistency matters :+1: